src/Widgets/DataImageWidget/DataImageWidgetComponent.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  provideComponent,
+  isInPlaceEditingActive,
+  LinkTag,
+  connect,
+  Link,
+  useData,
+  WidgetTag,
+  DataScope,
+  Obj,
+  ImageTag,
+  InPlaceEditingOff,
+} from 'scrivito'
+import { alignmentClassName } from '../../utils/alignmentClassName'
+import {
+  DataImageWidget,
+  DataImageWidgetInstance,
+} from './DataImageWidgetClass'
+import { isDataBinary } from '../../utils/dataBinaryToUrl'
+import { DataBinaryImage } from '../../Components/DataBinaryImage'
+
+provideComponent(DataImageWidget, ({ widget }) => {
+  const dataScope = useData()
```

Comment:
You don't need to use the data here and forward it to `ImageComponent` – the data can be used in `ImageComponent` directly.